### DISCUSSION TITLE
Meeting: April 9, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#99] [Announcements]
**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).

![image](https://github.com/OAI/sig-moonwalk/assets/54582/09ac7e12-59cd-4061-a7ec-1e36bb6387a7)

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Can we make links successful? | SIG members | |
Identity location ADR (add link) | SIG members | |

**Comment by mikekistler** [comment]
https://github.com/OAI/sig-moonwalk/discussions/100

**Comment by earth2marsh** [comment]
Video: https://zoom.us/rec/share/rKhRi4XC1-NSwennNLkPE1eJnQEUbPyQOT4PSazqZnqVd9Z3oi4TeAvBHZGwdxcm.CBdTRsJxKjznhWcK
